### PR TITLE
Add very bare-bones `process::Command`

### PR DESCRIPTION
**Pull request by tbu-**

### CODE REVIEW

**Comment by tbu-** [comment]
@homunkulus try

**Comment by homunkulus** [comment]
:hourglass: Trying commit afb395b with merge afb395b...

**Comment by homunkulus** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/japaric/steed/builds/196002801)
State: approved= try=True

File: src/sys/linux/pipe.rs [hunk 1]
```diff
@@ -2,4 +2,63 @@
...
+////////////////////////////////////////////////////////////////////////////////
+// Anonymous pipes
+////////////////////////////////////////////////////////////////////////////////
+
+pub struct AnonPipe(FileDesc);
+
+pub fn anon_pipe() -> io::Result<(AnonPipe, AnonPipe)> {
+    let mut fds = [0; 2];
+
+    // Unfortunately the only known way right now to create atomically set the
```
Comment (aleksander) [inline src/sys/linux/pipe.rs:23]: I saw similar comment in stdlib/sys/unix/net.rs. weak!() macro from sys/unix/weak.rs is used there. Can those macro be used here?

Comment (tbu-) [reply src/sys/linux/pipe.rs:23]: No. `weak!` tries to dynamically bind to symbols provided by the libc, which we do not have. Try to call the syscall directly, it'll return `-ENOSYS` if it doesn't exist on the host platform.

**Comment by japaric** [comment]
Thanks, @tbu-. Could you add a comment to #11 listing what needs to be done to reach parity with `std`'s Command and if anything is blocked by other issue (like `std::env`)?

@homunkulus r+

**Comment by homunkulus** [comment]
:pushpin: Commit 90fce3c has been approved by `japaric`

**Comment by japaric** [comment]
I really have to investigate why homu is getting stuck with `try`ed PRs...

Merging manually since `try` passed.

**Pull request merged**